assignments/arrays.js

```diff
@@ -63,31 +63,55 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+ for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i].id == 33) {
+        console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}`);
+    }
+}
+
```

Comment:
Two things:
1) The equality check could three equal signs instead of two. The difference between them is that three equal signs will not only check that the values are the same, but it will also check to make sure the value types are the same. (ie. they're both numbers). Using two equal signs would return true if a string "33" was compared against a number 33. This may be desired in some scenarios, but most of the time, it's recommended and good practice to make use of the strict equality check (`===`).

2) The loop continues to run even after the if statement passes and logs to the console. There's no need to continue the loop afterwards since it just uses up unnecessary resources. To "break" out of the loop, you could use the `break` keyword right after the console log:

```js
if (inventory[i].id == 33) {
  console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}`);
  break;
}
```